backend/middleware/check_auth.py

```
@@ -0,0 +1,22 @@
+# Third-party imports
+from fastapi import HTTPException, Depends, status
+from fastapi.security import OAuth2AuthorizationCodeBearer
+from backend.kindle_auth import kinde_client, kinde_configuration
```

Ruff

`backend.kindle_auth.kinde_configuration` imported but unused

Remove unused import: `backend.kindle_auth.kinde_configuration`

(F401)